src/app/auth/components/logout-link/logout-link.component.ts

```diff
@@ -8,12 +9,11 @@ import {Component, OnInit} from '@angular/core';
 	]
 })
 export class LogoutLinkComponent implements OnInit {
-
-	constructor() {
-	}
+	@Input() public setAuthStatus: (boolean) => void = _.noop;
```

Comment:
Use @Output for callbacks. In that case you do not need to bind the function to `this`